gnirehtet.cmd

```diff
@@ -0,0 +1,150 @@
+@ECHO OFF
+CALL :locate_file ADB %~dp0adb.exe "%LOCALAPPDATA%\Android\sdk\platform-tools\adb.exe"
```

Comment:
There isn't a system var for adb in windows. I'm searching adb in the same directory or in default path of Android Studio.